src/connectors/hits-per-page/connectHitsPerPage.ts

```diff
@@ -58,9 +58,7 @@ export type HitsPerPageConnectorParams = {
   /**
    * Function to transform the items passed to the templates.
    */
-  transformItems?: <TItem extends HitsPerPageConnectorParamsItem>(
-    objects: TItem[]
-  ) => TItem[];
+  transformItems?: TransformItems<HitsPerPageConnectorParamsItem>;
```

Comment:
this now no longer allows the user to pass another type which has more values since it's no longer generic. I don't think that's a big deal, but something to keep in mind.